cypress/integration/InstancesTest.ts

```diff
@@ -12,41 +12,52 @@ describe("Instances Test", () => {
       );
     });
 
-    it.skip("Submit", () => {
+    it("Submit", () => {
       const newInstanceName: string = "Some new instance";
       cy.ouiaId("create-smart-event-instance", "PF4/Button").click();
-      cy.ouiaId("create-instance", "PF4/ModalContent")
-        .should("be.visible")
-        .within(() => {
-          cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
-          cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
-            "have.text",
-            "Info alert:Your Smart Events instance will be ready for use shortly after creation."
-          );
-          cy.ouiaId("submit", "PF4/Button").click();
-        })
-        .should("not.exist");
-      //TODO: MGDOBR-706 - uncomment code for the verification that the instance was created (currently mocked server does not support it)
-      /*
+      cy.ouiaId("create-instance", "PF4/ModalContent").then(($modal) => {
+        cy.wrap($modal)
+          .should("be.visible")
+          .within(() => {
+            cy.ouiaId("new-name", "PF4/TextInput").type(newInstanceName);
+            cy.ouiaId("info-instance-available-soon", "PF4/Alert").should(
+              "have.text",
+              "Info alert:Your Smart Events instance will be ready for use shortly after creation."
+            );
+            cy.ouiaId("submit", "PF4/Button").click();
+          });
+        cy.wrap($modal, { timeout: 20000 }).should("not.exist");
```

Comment:
Why do we need to wait that long for the modal to close? The request to create an instance should be quick.